packages/keystone/KSv5v6/v5/prepareKeystone.js

```diff
@@ -38,6 +39,8 @@ function prepareKeystone ({ onConnect, extendExpressApp, schemas, schemasPreproc
     if (tasks && typeof tasks !== 'function') throw new Error('tasks should be a function like `() => [ Task ]`')
     if (apps && typeof apps !== 'function') throw new Error('apps should be a function like `() => [ App | Middleware ]`')
 
+    _internalGetExecutionContextAsyncLocalStorage().enterWith( { prepareKeystone: { id: v4(), argv: process.argv, execArgv: process.execArgv } })
```

Comment:
it's better to move it to [executionContext.js](https://github.com/open-condo-software/condo/pull/3683/files#diff-2b515060d90d3c4b6d869ad52f19005af586aa47dfbdf4973b349fd39a359854)

Reply:
It's better to use `exec` key for that